src/main/resources/assets/securitycraft/lang/pt_br.json

```diff
@@ -0,0 +1,1198 @@
+	"messages.securitycraft:blockpocket.invalidBlock.rotation": "O pilar %1$s em que o Block Pocket Manager é girado incorretamente!",
+	"messages.securitycraft:blockpocket.invalidBlock": "Bloco inválido \"%2$s\" %1$s o Block Pocket Manager! (Esperado: \"%3$s\")",
+	"messages.securitycraft:blockpocket.blockInWay": "\"%2$s\" está bloqueando o espaço %1$s do Block Pocket Manager!",
+	"messages.securitycraft:blockpocket.unowned": "Você não possui o bloco \"%2$s\" %1$s do Block Pocket Manager!",
+	"messages.securitycraft:blockpocket.position.1": "%s",
+	"messages.securitycraft:blockpocket.position.2": "%1$s e %2$s",
+	"messages.securitycraft:blockpocket.position.3": "%1$s, %2$s e %3$s",
+	"messages.securitycraft:blockpocket.position.blocksLeft": "%s Bloco (s) esquerdo de",
+	"messages.securitycraft:blockpocket.position.blocksRight": "%s Bloco (s) direito de",
+	"messages.securitycraft:blockpocket.position.blocksBehind": "%s Bloco (s) atrás",
+	"messages.securitycraft:blockpocket.position.blocksAbove": "%s Bloco (s) acima",
+	"messages.securitycraft:blockpocket.notEnoughItems": "Você não tem itens suficientes para montar uma bolsa de bloco desse tamanho!",
+	"messages.securitycraft:blockpocket.activated": "Block Pocket ativado!",
+	"messages.securitycraft:blockpocket.deactivated": "Block Pocket Desativado!",
+	"messages.securitycraft:blockpocket.assembled": "Bloqueio de bolso montado!",
+	"messages.securitycraft:blockpocket.assemblyFailed": "A montagem do bolso do bloco falhou %1 $ s o gerente de bolso de bloco.(Bloco incorreto: %2$s)",
+	"messages.securitycraft:ownable.validate": "O proprietário deste bloco foi validado com sucesso.",
+	"messages.securitycraft:ownable.ownerNotValidated": "O proprietário deste bloco foi alterado pelo trocador de proprietários da Universal e, portanto, precisa ser validado pelo novo proprietário para trabalhar novamente.Para validar um bloco, o novo proprietário do bloco precisa clicar com direito.",
+	"messages.securitycraft:sonic_security_system.notLinked": "Ainda não está vinculado a um bloco.Clique com o botão direito do mouse em um bloco de segurança compatível com este item em sua mão primeiro.",
+	"messages.securitycraft:sonic_security_system.blockLinked": "Ligado ao %1$s no %2$s.",
+	"messages.securitycraft:sonic_security_system.blockUnlinked": "Desvinculado do the %1$noat %2$s.",
+	"messages.securitycraft:sonic_security_system.linkMaxReached": "Você não pode vincular a blocos adicionais.Cada sistema de segurança sonora pode ser vinculado a um máximo de blocos de %s.",
+	"messages.securitycraft:sonic_security_system.locked": "Esse %s está sendo bloqueado por um sistema de segurança sonora!",
+	"messages.securitycraft:portable_tune_player.tune_saved": "Música salva.",
+	"messages.securitycraft:portable_tune_player.no_tune": "Nenhuma música disponível.",
+	"messages.securitycraft:sc_help": "%s: %s + %s",
+	"messages.securitycraft:formattedCoordinates": "X: %1$s Y: %2$s Z: %3$s",

```

Comment:
These lines can be removed, as they are the same in the english language file.

Reply:
I think it's good to keep it, as it will facilitate in the future a comparison between the files, done with WinMerge